gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -23,6 +23,7 @@
 public class BaselineErrorProneExtension {
     private static final ImmutableList<String> DEFAULT_PATCH_CHECKS = ImmutableList.of(
             // Baseline checks
+            "CatchBlockLogException",
```

Comment:
Low risk to fix this by default because existing baseline consumers pass this check. We don't attempt to fix checks that have been opted out of.